version/1/4/mk/code_of_conduct.md

```diff
@@ -0,0 +1,47 @@
+# Пакт за Придонесување - Kодекс на Oднесување
+
+## Нашиот завет
+
+Во интерес на негувањето на отворена и пријатна околина, ние како придонесувачи и одржувачи ветуваме дека ќе го правиме учеството во нашиот проект и нашата заедница пријатно искуство за сите, без вознемирување, без разлика на возраст, големина на телото, инвалидитет, етничка група, родов идентитет и претставување, ниво на искуство, националност, личен изглед, раса, религија, или сексуален идентитет и ориентација.
+
+## Нашите стандарди
```

Comment:
Hm, I think writing it "Наш" instead of "Нашиот" makes it more abstract and it's not very organic to use it like that in the Macedonian language. Take for instance that "Our mission" translates to "Нашата мисија", "Our vision" to "Нашата визија" etc. I think to show that this is really what are standards are, or what we vow to do (just noticed you deleted that comment), we can't go with "Наш" only. Doesn't it sound strange to you when you read it that way? What do you think?